src/agri_ai/line_bot/message_handler.py

```
@@ -0,0 +1,244 @@
+Message handler for LINE Bot.
+"""
+
+import asyncio
+import logging
+from typing import Dict, Any, Optional
+from datetime import datetime
+
+from linebot import LineBotApi
+from linebot.exceptions import LineBotApiError
+from linebot.models import (
+    MessageEvent, TextMessage, TextSendMessage,
```

Ruff (0.11.9)

`linebot.models.TextMessage` imported but unused

Remove unused import: `linebot.models.TextMessage`

(F401)